schema/annotation-source.yaml

```diff
@@ -185,7 +190,23 @@ $defs:
         enum: [ "supports", "uncertain", "opposes" ]
         description: >-
           The direction of this statement with respect to the target proposition.
-
+      has_evidence_of_type: 
+        type: string
+        enum: [ "case-control evidence", "clinical testing evidence", "curation-based evidence", 
+        "author statement from literature", "phenotyping evidence", "provider interpretation evidence", 
+        "reference population evidence", "research evidence" ]
+        description: >-
+          A term describing a type of evidence used to assess the validity of Statement's proposition.          
+      statement_quality: 
+        extends: information_quality
+        type: string
+        enum: ["4-star, practice guideline", "3-star, reviewed by expert panel", "1-star, criteria provided, 
+        single submitter", "0-star, no assertion criteria provided", "0-star, no assertion provided", 
+        "0-star, no interpretation for the single variant" ]
```

Comment:
Here we would simply concatenate the two parts of a review status rating to define a set of allowable statuses for SCVs.

Reply:
I don't want to have to parse these values apart. I'm not sure I will put the x-star values in at all as it is mostly a presentation thing.